### ISSUE TITLE
package_info_plus for windows returns invalid string

### ISSUE THREAD

**Issue by Farer** [fluttercommunity/plus_plugins#182]
![image](https://user-images.githubusercontent.com/931728/110768969-0a57cc80-829b-11eb-93a9-554948550eb6.png)

As you see above it returns invalid string.
I guessed that is blank string.
Buy it's not blank.
So I can't parse it as int.
Could you fix it ?

**Comment by p3root** [comment]
It also returns always just "1.0.0 ". I have a different version number in my pubspec file

**Comment by narumishi** [comment]
The last char is `\x000`. So should all values be trimmed in `getValue`?
https://github.com/fluttercommunity/plus_plugins/blob/2085d1c80c141efb488d841644281e3e7df03683/packages/package_info_plus_windows/lib/src/file_version_info.dart#L48
can anyone have a quick fix on it?

BTW @p3root flutter hasn't implement generating version for `exe` from pubspec yet, it's not an issue of this plugin

**Comment by p3root** [comment]
Any workaroung for that?

**Comment by narumishi** [comment]
@p3root please tracking flutter/flutter#73652

**Comment by narumishi** [comment]
@p3root if you want to update exe version, you can edit `windows/runner/Runner.rc`. Just change the default value for VERSION_AS_NUMBER and VERSION_AS_STRING. This is my current solution before official support. You will need to change it manually every time changing version.
```
#ifdef FLUTTER_BUILD_NUMBER
#define VERSION_AS_NUMBER FLUTTER_BUILD_NUMBER
#else
#define VERSION_AS_NUMBER 1,3,8,33
#endif

#ifdef FLUTTER_BUILD_NAME
#define VERSION_AS_STRING #FLUTTER_BUILD_NAME
#else
#define VERSION_AS_STRING "1.3.8.33"
#endif
```
The original should be `x.y.z` and `"x.y.z"`, I add build number after. If you want to use `package_info_plus` to parse it, you should change `1.3.8.33` to `1.3.8+33` in my example.

**Comment by mhadaily** [comment]
duplicate https://github.com/fluttercommunity/plus_plugins/issues/343

reason: https://github.com/flutter/flutter/issues/73652

**Comment by fabiancrx** [comment]
This issue could be closed and continue tracking at #343 as @mhadaily  stated.
